runtime/common/src/slots.rs

```diff
@@ -44,257 +43,70 @@ pub trait Config: frame_system::Config {
-#[cfg_attr(feature = "std", derive(Debug))]
-pub enum Bidder<AccountId> {
-	/// An account ID, funds coming from that account.
-	New(NewBidder<AccountId>),
-
-	/// An existing parachain, funds coming from the amount locked as part of a previous bid topped
-	/// up with funds administered by the parachain.
-	Existing(ParaId),
-}
-
-impl<AccountId: Clone + Default + Codec> Bidder<AccountId> {
-	/// Get the account that will fund this bid.
-	fn funding_account(&self) -> AccountId {
-		match self {
-			Bidder::New(new_bidder) => new_bidder.who.clone(),
-			Bidder::Existing(para_id) => para_id.into_account(),
-		}
-	}
-}
-
-/// Information regarding a parachain that will be deployed.
-///
-/// We store either the bidder that will be able to set the final deployment information or the
-/// information itself.
-#[derive(Clone, Eq, PartialEq, Encode, Decode)]
-#[cfg_attr(feature = "std", derive(Debug))]
-pub enum IncomingParachain<AccountId, Hash> {
-	/// Deploy information not yet set; just the bidder identity.
-	Unset(NewBidder<AccountId>),
-	/// Deploy information set only by code hash; so we store the code hash, code size, and head data.
-	///
-	/// The code size must be included so that checks against a maximum code size
-	/// can be done. If the size of the preimage of the code hash does not match
-	/// the given code size, it will not be possible to register the parachain.
-	Fixed { code_hash: Hash, code_size: u32, initial_head_data: HeadData },
-	/// Deploy information fully set; so we store the code and head data.
-	Deploy { code: ValidationCode, initial_head_data: HeadData },
 }
 
 type LeasePeriodOf<T> = <T as frame_system::Config>::BlockNumber;
-// Winning data type. This encodes the top bidders of each range together with their bid.
-type WinningData<T> =
-	[Option<(Bidder<<T as frame_system::Config>::AccountId>, BalanceOf<T>)>; SLOT_RANGE_COUNT];
-// Winners data type. This encodes each of the final winners of a parachain auction, the parachain
-// index assigned to them, their winning bid and the range that they won.
-type WinnersData<T> =
-	Vec<(Option<NewBidder<<T as frame_system::Config>::AccountId>>, ParaId, BalanceOf<T>, SlotRange)>;
 
 // This module's storage items.
 decl_storage! {
 	trait Store for Module<T: Config> as Slots {
-		/// The number of auctions that have been started so far.
-		pub AuctionCounter get(fn auction_counter): AuctionIndex;
-
-		/// Ordered list of all `ParaId` values that are managed by this module. This includes
-		/// chains that are not yet deployed (but have won an auction in the future).
-		pub ManagedIds get(fn managed_ids): Vec<ParaId>;
+		/// Amount held on deposit for each para and the original depositor.
+		///
+		/// The given account ID is responsible for registering the code and initial head data, but may only do
+		/// so if it isn't yet registered. (After that, it's up to governance to do so.)
+		pub Paras: map hasher(twox_64_concat) ParaId => Option<(T::AccountId, BalanceOf<T>)>;
```

Comment:
this storage seems only used by crownloans, what is the reason to have it here ?
Also it seems a bit duplication of `paras_registrar::Paras` but I don't have the full picture yet.